en/components/list.md

```diff
@@ -5,71 +5,368 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
 ---
 
 ##List View
+
 <p class="highlight">The Ignite UI for Angular List component displays rows of items and supports one or more header items as well as search and filtering of list items. Each list item is completely templateable and will support any valid HTML or Angular component. </p>
 <div class="divider"></div>
 
 ### List Demo
-<div class="sample-container" style="height: 760px">
-<iframe src='https://{environment:host}/angular-demos/list' width="100%" height="100%" seamless frameBorder="0"></inframe>
+
+<div class="sample-container" style="height: 477px">
+<iframe src='http://{environment:host}/list-sample-4' width="100%" height="100%" seamless frameBorder="0"></inframe>
```

Comment:
`</inframe>` should be `</iframe>`. There's other occurrences down the document.